mne_bids/write.py

```diff
@@ -2565,7 +2564,7 @@ def mark_channels(bids_path, *, ch_names, status, descriptions=None, verbose=Non
         idx = tsv_data["name"].index(ch_name)
         logger.info(
             f"Processing channel {ch_name}:\n"
-            f"    status: bad\n"
+            f"    status: {status_}\n"
```

Comment:
no idea why this was just `bad` before 🤔  especially because the column is initialized with all `good` if it wasn't present before.

```
    # Read sidecar and create required columns if they do not exist.
    if "status" not in tsv_data:
        logger.info('No "status" column found in input file. Creating.')
        tsv_data["status"] = ["good"] * len(tsv_data["name"])
```

I think this is also why in the docstring it said something about "default is 'bad'". If we start writing the status for a few channels only, the status for all other channels gets automatically set. However it gets set to "good", not "bad", ... which I find reasonable, but which was misleading in the function docstr.

Reply:
ok, I followed up on this in https://github.com/mne-tools/mne-bids/pull/1293/commits/5c042796f634e96c220682328de7968082c7b874